ac_s.lua

```diff
@@ -850,6 +906,23 @@ Citizen.CreateThread(function()
 			end
 		end
 	end)
+		
+	AddEventHandler("clearPedTasksEvent", function(ped, source, data)
+    		if data.immediately then
+        		if IsPedAPlayer((ped)) then
```

Comment:
wouldnt this trigger if i call Clear Tasks on myself? or does the event not take that into account

Reply:
if think now what i have done is good to use